README.md

```diff
@@ -1647,6 +1647,7 @@ A [proxy](https://en.wikipedia.org/wiki/Proxy_server) is a server application th
 Software and tools for managing [recipes](https://en.wikipedia.org/wiki/Recipe).
 
 - [Bar Assistant](https://github.com/karlomikus/bar-assistant) - Bar assistant is a self hosted application for managing your home bar. It allows you to add your ingredients, search for cocktails and create custom cocktail recipes. ([Demo](https://bar.karlomikus.com/)) `MIT` `PHP/Docker`
+- [Chowdown.io](https://chowdown.io/) - Simple recipes in Markdown format ([Demo](https://chowdown.io/), [Source Code](https://github.com/clarklab/chowdown)) `Unlicense` `PHP`
```

Comment:
```suggestion
- [Chowdown.io](https://chowdown.io/) - Simple recipes in Markdown format. ([Source Code](https://github.com/clarklab/chowdown)) `Unlicense` `Ruby`
```

- The homepage contents and the demo site contents are actually the same. The demo link is not needed
- This is not PHP, the main requirement is actually Jekyll/Ruby
>This is a Jekyll build. Make sure you have Jekyll [installed](https://jekyllrb.com/). To install, run this command: gem install bundler jekyll

Reply:
Oof, totally missed that